src/lib.rs

```diff
@@ -96,6 +116,15 @@ pub struct BatchLCProof<F: Field, PC: PolynomialCommitment<F>> {
     pub evals: Option<Vec<F>>,
 }
 
+impl<F: Field, PC: PolynomialCommitment<F>> Clone for BatchLCProof<F, PC> {
+    fn clone(&self) -> Self {
+        BatchLCProof {
+            proof: self.proof.clone(),
+            evals: self.evals.clone(),
+        }
+    }
+}
```

Comment:
Can you derive Clone? Either via the standard macro, or via the `derivative` macro?